README.md

```diff
@@ -104,6 +103,46 @@ The following Visual Studio Code settings are available for the Nim extension.
 }
 ```
 
+#### Nim Lanugage Server integration (experimental)
+
+`vscode-nim` provides optional integration with [Nim Language Server](https://github.com/Nim-lang/langserver) as an
+alternative of using `nimsuggest`. In order to activate it, put the following
+lines in your config:
+
+```json
+{
+    "nim.enableLsp": true
+}
+```
+
+Similar to `vscode-nim` [Nim Language Server](https://github.com/Nim-lang/langserver) uses `nimsuggest` under the
+hood and to control the way `nimsuggest` is created you can use
```

Comment:
Why isn't this using the existing project and project mapping config?

Reply:
TBO I was not aware of this config. I will check both can be adjusted to use the same set of settings.

Reply:
Nailed it